### ISSUE TITLE
[Feature request - CalDAV] Events notifications are not synchronized 

### ISSUE THREAD

**Issue by pchampio** [marcoancona/MineTime#163]
Using the CalDAV(beta) calendar source, the notifications aren't synchronized with the remote account.
Do you have any plan on adding this feature?

**Comment by marcoancona** [comment]
Yes, at the moment notification settings are not synchronized with any provider. This is on our backlog.
